cmd/tests/cmd_test.go

```diff
@@ -1038,23 +1065,24 @@ func TestAbortedByScriptAbortInTeardown(t *testing.T) {
 }
 
 func testAbortedByScriptTestAbort(
-	t *testing.T, shouldHaveMetrics bool, script []byte, runTest func(*testing.T, *globalTestState),
-) *globalTestState {
+	//nolint: unparam  // will be fixed by https://github.com/grafana/k6/pull/2800
+	t *testing.T, shouldHaveMetrics bool, script []byte, runTest func(*testing.T, *state.GlobalTestState),
+) *state.GlobalTestState {
```

Comment:
@na-- There were two linter issues here.

- `unparam  testAbortedByScriptTestAbort - shouldHaveMetrics always receives true`
   Which I decided to silence until #2800 is merged, which does pass `false`.

- And the one that currently fails:
  `unparam  testAbortedByScriptTestAbort - result 0 (*go.k6.io/k6/cmd/state.GlobalTestState) is never used`

  Which I think is pointing out that `GlobalTestState.ExpectedExitCode` is always always `0` and never checked?

  I may be wrong, and in any case, I'm not sure how to fix it, or if we should silence it as well. Do you have an idea?

BTW, this is partly why I'm not a fan of overusing helper functions in tests, which I pointed out in [this PR](https://github.com/grafana/k6/pull/2800#discussion_r1048641706). They make tests more difficult to follow, and result in cryptic issues like this.

Reply:
I silenced the second error as well :man_shrugging: